src/Ztm.Data.Entity.Postgres/ByteaHandler.cs

```diff
@@ -9,26 +9,19 @@ namespace Ztm.Data.Entity.Postgres
 {
     class ByteaHandler : Npgsql.TypeHandlers.ByteaHandler, INpgsqlTypeHandler<uint256>
     {
-        async ValueTask<uint256> INpgsqlTypeHandler<uint256>.Read(NpgsqlReadBuffer buf, int len, bool async, FieldDescription fieldDescription)
+        async ValueTask<uint256> INpgsqlTypeHandler<uint256>.Read(NpgsqlReadBuffer buf, int len, bool async,
+            FieldDescription fieldDescription)
         {
-            try
-            {
-                var bytes = new byte[len];
-                await buf.ReadBytes(bytes, 0, len, async);
-                return new uint256(bytes, false);
-            }
-            catch (Exception e)
-            {
-                throw new NpgsqlSafeReadException(e);
-            }
+            var bytes = await Read<byte[]>(buf, len, async, fieldDescription);
```

Comment:
This one is quite dangerous due to this overload was designed to be able to read arbitrary type. We should use https://github.com/npgsql/npgsql/blob/dev/src/Npgsql/TypeHandlers/ByteaHandler.cs#L38 instead.